packages/control-property-editor/src/panels/info-center/InfoMessageItem.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from 'react';
+import type { ReactElement } from 'react';
+import { Stack, Text } from '@fluentui/react';
+import { UIMessageBar, UIDialog, UITextInput, UIIconButton, UiIcons } from '@sap-ux/ui-components';
+import { useDispatch } from 'react-redux';
+import type { InfoCenterItem } from '../../slice';
+import { MessageBarType } from '@sap-ux-private/control-property-editor-common';
+import { clearInfoCenterMessage } from '../../slice';
+import { useTranslation } from 'react-i18next';
+import './InfoMessageItem.scss';
+
+/**
+ * Returns the corresponding string representation of the message type.
+ *
+ * @param {MessageBarType} type - The type of the message.
+ * @returns {'error' | 'warning' | 'info'} The string representation of the message type.
+ */
+const getMessageType = (type: MessageBarType) => {
+    switch (type) {
+        case MessageBarType.error:
+            return 'error';
+        case MessageBarType.warning:
+            return 'warning';
+        default:
+            return 'info';
+    }
+};
+
+/**
+ * React element for info center item.
+ *
+ * @param item
+ * @returns ReactElement
+ */
+export function InfoMessageItem(item: Readonly<InfoCenterItem>): ReactElement {
+    const { t } = useTranslation();
+    const dispatch = useDispatch();
+    const [isExpanded, setIsExpanded] = useState<boolean>();
+    const [isRead, setIsRead] = useState<boolean>();
+    const [isOpenedModal, setIsOpenedModal] = useState<boolean>();
```

Comment:
To avoid unintended behavior these need to have default values which in our case are false.

```suggestion
    const [isExpanded, setIsExpanded] = useState<boolean>(false);
    const [isRead, setIsRead] = useState<boolean>(false);
    const [isOpenedModal, setIsOpenedModal] = useState<boolean>(false);
```